boxes/drawing.py

```diff
@@ -686,7 +686,7 @@ def finish(self, inner_corners="loop"):
                     elif C == "T":
                         m, text, params = c[3:]
                         tm = " ".join(f"{m[i]:.3f}" for i in (0, 3, 1, 4, 2, 5))
-                        text = text.replace("(", "r\(").replace(")", r"\)")
```

Comment:
Well, the new `r` is correct but actually the one in the double quotes is just at the wrong place.